### PR TITLE
Fix documentation build errors

### PR DESCRIPTION
**Pull request by JaydenR2305**
This pull request resolves an error in building documentation that seems to have arisen due to towncrier release 22.8.0.

- [x] I have added a changelog entry for this pull request.
- [x] If adding new functionality, I have added tests and
      docstrings.
- [x] I have fixed any newly failing tests.

### CODE REVIEW

**Comment by JaydenR2305** [comment]
@pheuer 🎊

File: requirements/docs.txt [hunk 1]
```diff
@@ -21,4 +21,4 @@ sphinx-notfound-page >= 0.8
 sphinx-reredirects
 sphinx_rtd_theme >= 1.0.0
 sphinxcontrib-bibtex
-towncrier >= 19.2.0
+towncrier == 21.9.0
```
Comment (namurphy) [inline requirements/docs.txt:24]:
```suggestion
towncrier >= 19.2.0, < 22.8.0
```

File: setup.cfg [hunk 2]
```diff
@@ -106,7 +106,7 @@ docs =
     sphinx-reredirects
     sphinx_rtd_theme >= 1.0.0
     sphinxcontrib-bibtex
-    towncrier >= 19.2.0
+    towncrier == 21.9.0
```
Comment (namurphy) [inline setup.cfg:109]:
```suggestion
    towncrier >= 19.2.0, < 22.8.0
```

**Comment by pheuer** [comment]
see discussion in #1703

**Review by namurphy — COMMENTED** [review]
Thank you for tracking down what was causing this so quickly!  I added some suggestions to broaden the range of towncrier versions to reduce the chances of a version conflict with the requirements of any other packages.  Thank you again!

**Review by namurphy — APPROVED** [review]

**Pull request merged**